api/service/syncing/syncing.go

```diff
@@ -929,6 +949,10 @@ func (ss *StateSync) SyncLoop(bc *core.BlockChain, worker *worker.Worker, isBeac
 					isBeacon, bc.ShardID(), otherHeight, currentHeight)
 			break
 		}
+		if outOfSyncCount < syncStatusCheckCount {
```

Comment:
why it will affect sync performance? the sync check happens every 60s. For each of the check, to avoid false positive, better to double check the status before entering sync mode. And doing 3 seconds of double check shoudn't really matter if the node is really out of sync for 60s.

Reply:
Sync happens in two case: 1. triggered every 60 seconds 2. triggered by consensus. The delay is not good for the second case. I believe currently it is taking too long to finish a sync. There is already 3s sleep in getMaxPeerHeight, so it is likely to take more than extra 9s to do one batch sync, which is terrible. 

Also these two sync will block each other. So I would suggest to wait extra 3 second for the first trigger, and do not wait extra time for the second case.

Reply:
sure. I will remove this.

Reply:
BTW, ticker removed in this PR: https://github.com/harmony-one/harmony/pull/3436